### PR TITLE
 Translate snap article (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Translation of #1888
Ref #1894 

Actual diff is bfb9042
Please review this @ruby/www-ruby-lang-org-i18n-ko :pray:

1 question: Is it better to translate `snap`...?

### CODE REVIEW

File: ko/documentation/installation/index.md [hunk 1]
```diff
@@ -94,6 +95,24 @@ $ sudo yum install ruby
 설치된 버전은 보통 그 특정 배포 버전 릴리스 시기의 최신 버전의 루비입니다.
 
 
+### snap (Ubuntu나 다른 리눅스 배포판)
+{: #snap}
+
+Snap은 Canonical에 의해 개발된 패키지 매니저입니다. 이는 Ubuntu에서 바로 사용가능합니다만, 다른 리눅스 배포판에서도 동작합니다.
```
Comment (yous) [inline ko/documentation/installation/index.md]:
```suggestion
Snap은 Canonical에 의해 개발된 패키지 매니저입니다. 이는 Ubuntu에서 바로 사용 가능합니다만, 다른 리눅스 배포판에서도 동작합니다.
```

File: ko/news/_posts/2018-11-08-snap.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+여러 채널을 동시에 사용할 수도 있습니다. 다음 명령을 통해 루비 2.3으로 전환할 수 있습니다.
+
+```
+sudo snap switch ruby --channel=2.3/stable
+sudo snap refresh
+```
+
+루비 snap은 `GEM_HOME`과 `GEM_PATH` 환경 변수를 `$HOME/.gem`으로
+지정합니다.
+그러므로 만약 `rails`나 `rspec`과 같은 설치 젬의 명령을 `bundle exec` 없이
```
Comment (yous) [inline ko/news/_posts/2018-11-08-snap.md]:
```suggestion
그러므로 만약 젬에서 설치한 `rails`나 `rspec`과 같은 명령을 `bundle exec` 없이
```

File: ko/news/_posts/2018-11-08-snap.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+실행하고 싶다면 아래의 명령을 쉘 rc 파일(예를 들어 `.bashrc`)에 추가해야
```
Comment (yous) [inline ko/news/_posts/2018-11-08-snap.md]: 쉘 → 셸 https://ko.dict.naver.com/#/entry/koko/4e8659cbc50f42599d34c064189f8f62
```suggestion
실행하고 싶다면 아래의 명령을 셸 rc 파일(예를 들어 `.bashrc`)에 추가해야
```

**Comment by riseshia** [comment]
Applied reviews in e1e2bcd

**Comment by riseshia** [comment]
Thanks for review!! XD

**Pull request merged**